router/package_request.go

```diff
@@ -171,14 +171,20 @@ func RespondToPackageRequest(
 				packageRequest.GithubTree,
 			)
 
-			res.Header().Set(httpContentTypeHeader, contentTypeHTML)
-			res.Write([]byte(generateGoGetMetadata(
-				packageRequest.Author,
-				packageRequest.Repo,
+			packageModel := models.PackageModel{Author: &packageRequest.Author, Repo: &packageRequest.Repo}
```

Comment:
I'm not sure if this is the correct place but after testing it seemed to be? I figured the process would look very close to this. The sub-versioner will do a look up if the packageModel has been versioned before or it will go through the versioning process.